pandas/_libs/src/inference.pyx

```diff
@@ -165,20 +165,8 @@ cdef class Seen(object):
                      two conflict cases was also detected. However, we are
                      trying to force conversion to a numeric dtype.
         """
-        if self.uint_ and (self.null_ or self.sint_):
-            if not self.coerce_numeric:
-                return True
-
-            if self.null_:
-                msg = ("uint64 array detected, and such an "
```

Comment:
interesting, so these cases were not actually tested then? (IOW you didn't have to fix any tests)?

Reply:
nvm. I see you blew away the entire section and re-wrote.